libs/dataset-explorer/src/lib/TableView/TableView.tsx

```diff
@@ -0,0 +1,294 @@
+          }
+        }
+      }
+      this.props.onAllSelectedItemsChange(this.selection.getSelection());
+      this.props.telemetryHook?.({
+        level: TelemetryLevels.ButtonClick,
+        type: TelemetryEventName.IndividualFeatureImportanceSelectedDatapointsUpdated
+      });
+    }
+  });
+
+  public constructor(props: ITableViewProps) {
+    super(props);
+
+    const tableState = this.updateItems();
+
+    this.state = {
+      indexToUnselect: undefined,
+      selectedIndices: [],
+      ...tableState
+    };
+  }
+
+  public componentDidUpdate(prevProps: ITableViewProps): void {
+    if (this.props.selectedCohort !== prevProps.selectedCohort) {
+      this.setState(this.updateItems());
+    }
+    if (this.state.indexToUnselect) {
+      this.selection.toggleIndexSelected(this.state.indexToUnselect);
+      this.setState({ indexToUnselect: undefined });
+    }
+  }
+
+  public render(): React.ReactNode {
+    if (this.state.rows === undefined || this.state.columns === undefined) {
+      return React.Fragment;
+    }
+    const hasTextImportances =
+      !!this.context.modelExplanationData?.precomputedExplanations
+        ?.textFeatureImportance;
+    const classNames = tableViewStyles();
+    return (
+      <Stack tokens={{ padding: "l1" }}>
+        <Stack.Item className={classNames.infoWithText}>
+          <Text variant="medium">
+            {localization.ModelAssessment.FeatureImportances.IndividualFeature}
+          </Text>
+        </Stack.Item>
+        <Stack.Item className={classNames.selectionCounter}>
+          <LabelWithCallout
+            label={localization.formatString(
+              localization.ModelAssessment.FeatureImportances.SelectionCounter,
+              this.selection.count,
+              this.maxSelectable
+            )}
+            calloutTitle={undefined}
+            renderOnNewLayer
+            type="label"
+          >
+            <Text block>
+              {localization.ModelAssessment.FeatureImportances.SelectionLimit}
+            </Text>
+          </LabelWithCallout>
+        </Stack.Item>
+        <Stack.Item className={classNames.tabularDataView}>
+          <div style={{ height: "500px", position: "relative" }}>
+            <Fabric>
+              <ScrollablePane scrollbarVisibility={ScrollbarVisibility.auto}>
+                <MarqueeSelection selection={this.selection}>
+                  <DetailsList
+                    items={this.state.rows}
+                    columns={this.state.columns}
+                    groups={this.state.groups}
+                    setKey="set"
+                    layoutMode={DetailsListLayoutMode.fixedColumns}
+                    constrainMode={ConstrainMode.unconstrained}
+                    onRenderDetailsHeader={this.onRenderDetailsHeader}
+                    selectionPreservedOnEmptyClick
+                    ariaLabelForSelectionColumn="Toggle selection"
+                    checkButtonAriaLabel="Row checkbox"
```

Comment:
wondering if we need to localize aria label?

Reply:
done, localized just in case both aria labels